tasks/deploy-redis.js

```diff
@@ -47,21 +44,38 @@ async function deployRedis(config, rev) {
   gutil.log(
     gutil.colors.yellow(env()),
     'Uploading revision',
-    gutil.colors.green(rev),
+    gutil.colors.green(rev)
   );
 
   await Promise.all(
     (config.files || [config]).map(async fileConfig => {
-      const fileName = fileConfig.indexPath;
-      gutil.log(
-        gutil.colors.yellow(env()),
-        'Deploying',
-        gutil.colors.magenta(fileName),
-        '...',
+      const filesToUpload = ['index', 'info'].reduce((acc, v) => {
```

Comment:
It seems to me like we're moving parts of the resulting key path away from the config and into the codebase itself. I think we should strive for configurability here and make this configurable similarly to what we do e.g. here https://github.com/productboard/webpack-deploy/pull/45/files#diff-65822c82e442afc2b34df09c3ad934a0R37